packages/grafana-toolkit/src/cli/tasks/package.build.ts

```diff
@@ -81,26 +72,22 @@ const buildTaskRunner: TaskRunner<PackageBuildOptions> = async ({ scope }) => {
     throw new Error('Provide packages with -s, --scope <packages>');
   }
 
-  const scopes = scope.split(',').map(s => {
-    return async () => {
-      cwd = path.resolve(__dirname, `../../../../grafana-${s}`);
-      // Lerna executes this in package's dir context, but for testing purposes I want to be able to run from root:
-      // grafana-toolkit package:build --scope=<package>
-      process.chdir(cwd);
-      distDir = `${cwd}/dist`;
-      const pkg = require(`${cwd}/package.json`);
-      console.log(chalk.yellow(`Building ${pkg.name} (package.json version: ${pkg.version})`));
-
-      await clean();
-      await compile();
-      await moveStaticFiles(pkg);
-      await bundle();
-      await preparePackage(pkg);
-      await moveFiles();
-    };
+  const scopes = scope.split(',').map(async s => {
+    const repoRoot = path.resolve(__dirname, '../../../../../');
+    const packageRoot = `${repoRoot}/packages/grafana-${s}`;
+    const relativePackageRoot = path.relative(repoRoot, packageRoot);
```

Comment:
Isn't this cleaner, than going via `path.relative`?
```suggestion
    const relativePackageRoot = `packages/grafana-${s}`;
    const packageRoot = `${repoRoot}/${relativePackageRoot}`;
```

Reply:
Yup. My brain was tired.